### PR TITLE
PHP Scale Codec and API RFP

### PR DESCRIPTION
**Pull request by Swader**
PHP is extremely popular. It makes no sense to not have a library for communicating with Substrate from it.

### CODE REVIEW

**Comment by Noc2** [comment]
Looks good. I just have one comment: Could you integrate the “Status: open”? (see https://github.com/w3f/General-Grants-Program/blob/master/rfp-proposal/suggestion-template.md#title-of-the-rfp-proposal) I recently added this so that people are aware of which rfps are still open and which are already done.

File: rfp-proposal/php-api.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# PHP Substrate API
+
+* **Proposer:** [swader](https://github.com/api)
```
Comment (Swader) [inline rfp-proposal/php-api.md:3]:
```suggestion
* **Proposer:** [swader](https://github.com/api)
* **Status:** Open
```

File: rfp-proposal/php-scale.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# PHP Version of SCALE Codec
+
+* **Proposer:** [swader](https://github.com/swader)
```
Comment (Swader) [inline rfp-proposal/php-scale.md:3]:
```suggestion
* **Proposer:** [swader](https://github.com/swader)
* **Status:** Open
```

**Comment by Swader** [comment]
Done

**Comment by Noc2** [comment]
And merged ;-) Thanks!

**Comment by mmagician** [comment]
> PHP is extremely popular

Depends on how you look at it. While many websites might still be using it, my personal preference would be to shy away from a tech stack which still relies on PHP.
I think this is especially true for new web applications.
Also, in terms of popularity among developers, it has been pretty low in the rankings in the recent years.

**Comment by Swader** [comment]
> Depends on how you look at it

Objectively and statistically. The community is second only to JS.

> shy away from a tech stack which still relies on PHP

Why?

**Comment by mmagician** [comment]
> Objectively and statistically. The community is second only to JS.

My point of reference was https://insights.stackoverflow.com/survey/2020#technology-most-loved-dreaded-and-wanted-languages
Quick google search presents, as always, a ton of different views, so I'm sure in some ranking it's higher, whereas it's lower in others. The one above doesn't take community size into account, for example.

Though I must say my gut developer feeling underestimated the present popularity in some rankings.

I'm not saying I wouldn't support PHP developers building with Polkadot, but that I personally wouldn't go as far as **proposing** it be built.

**Pull request merged**